tests/protocol/test_hostplugin.py

```diff
@@ -536,6 +538,44 @@ def test_validate_page_blobs(self):
                         test_goal_state,
                         exp_method, exp_url, exp_data)
 
+    def test_validate_http_request_when_uploading_logs(self):
+        with mock_wire_protocol(DATA_FILE) as protocol:
+            test_goal_state = protocol.client._goal_state
```

Comment:
NIT: Why not use `protocol.client.get_goal_state()` instead.

Reply:
Thanks!